libraries/operation-graph/tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
-  "extends": "./node_modules/@rushstack/heft-node-rig/profiles/default/tsconfig-base.json",
+  "extends": "./node_modules/decoupled-local-node-rig/profiles/default/tsconfig-base.json",
 
   "compilerOptions": {
-    "isolatedModules": true,
+    // TODO: Consider turning this on for all projects, or removing it here
```

Comment:
Is this something we want to solve in this PR? Or wait til later?

Reply:
Later.